src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksBatchUtilityServiceImpl.java

```diff
@@ -211,7 +211,8 @@ public void registerKfsPvenDisapprovalForExistingPaymentWorksVendor(String kfsVe
             LOG.info("registerKfsPvenDisapprovalForExistingPaymentWorksVendor: Disapproving PaymentWorks originating Vendor with table ID '" + pmwVendor.getId()
                      + "' and kfsVendorDocumentNumber '" + kfsVendorDocumentNumber + "' and pmwVendorRequestId '" + pmwVendor.getPmwVendorRequestId() + "'");
             getPaymentWorksVendorDao().updateExistingPaymentWorksVendorInStagingTable(pmwVendor.getId(), PaymentWorksConstants.KFSVendorProcessingStatus.VENDOR_DISAPPROVED, vendorDetail.getVendorHeaderGeneratedIdentifier(),
-                                                                                      vendorDetail.getVendorDetailAssignedIdentifier(), getDateTimeService().getCurrentTimestamp());
+                                                                                      vendorDetail.getVendorDetailAssignedIdentifier(), PaymentWorksConstants.SupplierUploadStatus.INELIGIBLE_FOR_UPLOAD,
+                                                                                      getDateTimeService().getCurrentTimestamp());
         } else {
```

Comment:
As you previously noted, PaymentWorksConstants.SupplierUploadStatus.INELIGIBLE_FOR_UPLOAD should be removed.